src/vpic/dump.cc

```diff
@@ -12,6 +12,13 @@
 #include "vpic.h"
 #include "../util/io/FileUtils.h"
 
+#ifdef VPIC_ENABLE_HDF5
+#include "hdf5.h" // from the lib
+#endif
+
+/* 1 means convert cell index to global */
+#define OUTPUT_CONVERT_GLOBAL_ID 1
```

Comment:
I guess this needs to become a compile time option at some point

Reply:
Yes. For now I just reuse what is in the `hdf5` branch.

Reply:
If we're not fixing this now lets at least get a `//TODO:` or something on there